src/Controllers/SchemaController.js

```diff
@@ -330,26 +330,26 @@ const injectDefaultSchema = ({className, fields, classLevelPermissions, indexes}
 
 const _HooksSchema =  {className: "_Hooks", fields: defaultColumns._Hooks};
 const _GlobalConfigSchema = { className: "_GlobalConfig", fields: defaultColumns._GlobalConfig }
-const _PushStatusSchema = convertSchemaToAdapterSchema(injectDefaultSchema({
+const _PushStatusSchema = convertSchemaToAdapterSchema({
   className: "_PushStatus",
   fields: {},
   classLevelPermissions: {}
-}));
-const _JobStatusSchema = convertSchemaToAdapterSchema(injectDefaultSchema({
+});
+const _JobStatusSchema = convertSchemaToAdapterSchema({
   className: "_JobStatus",
   fields: {},
   classLevelPermissions: {}
-}));
-const _JobScheduleSchema = convertSchemaToAdapterSchema(injectDefaultSchema({
+});
+const _JobScheduleSchema = convertSchemaToAdapterSchema({
   className: "_JobSchedule",
   fields: {},
   classLevelPermissions: {}
-}));
-const _AudienceSchema = convertSchemaToAdapterSchema(injectDefaultSchema({
+});
+const _AudienceSchema = convertSchemaToAdapterSchema({
   className: "_Audience",
-  fields: defaultColumns._Audience,
```

Comment:
Inconsistent and not required as convertSchemaToAdapterSchema calls injectDefaultSchema which adds these fields anyway